src/app/main-overview/components/ControlPlaneDetails.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div class="stack">
+    <KCard>
+      <template #body>
+        <div class="card-header">
+          <div class="card-title">
+            <h2>{{ t('main-overview.detail.health.title') }}</h2>
+          </div>
+        </div>
+
+        <div class="variable-columns">
+          <ResourceStatus
+            :total="store.getters['config/getMulticlusterStatus'] ? props.zoneOverviews.length : 1"
+            data-testid="zone-control-planes-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-location-on.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.zone_control_planes') }}
+            </template>
+          </ResourceStatus>
+
+          <ResourceStatus
+            :total="meshInsight.meshesTotal"
+            data-testid="meshes-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-circles-ext.svg?url">
+            </template>
+
+            <template #title>
+              {{ t('main-overview.detail.health.meshes') }}
+            </template>
+          </ResourceStatus>
+
+          <ResourceStatus
+            :total="meshInsight.services.total"
+            data-testid="services-status"
+          >
+            <template #icon>
+              <img src="@/assets/images/icon-wifi-tethering.svg?url">
```

Comment:
just the icon from the design, weird though ;)

Reply:
Yes that's a good point too. What was the decision in the end?

Reply:
I kept those names. This icon in particular is currently used for both services and DPPs so I’m not sure what a better name would be.